coaster/examples/readme.rs

```diff
@@ -1,27 +1,47 @@
 extern crate coaster as co;
 extern crate coaster_nn as nn;
-use crate::co::prelude::*;
-use crate::nn::*;
+
+use co::prelude::*;
+use co::frameworks::native::flatbox::FlatBox;
+use nn::*;
 
 fn write_to_memory<T: Copy>(mem: &mut FlatBox, data: &[T]) {
-	let mem_buffer = mem.as_mut_slice::<T>();
-	for (index, datum) in data.iter().enumerate() {
-	    mem_buffer[index] = *datum;
-	}
+    let mem_buffer = mem.as_mut_slice::<T>();
+    for (index, datum) in data.iter().enumerate() {
+        mem_buffer[index] = *datum;
+    }
 }
 
+#[cfg(not(feature = "cuda"))]
 fn main() {
-    // Initialize a CUDA Backend.
     let backend = Backend::<Native>::default().unwrap();
     // Initialize two SharedTensors.
     let mut x = SharedTensor::<f32>::new(&(1, 1, 3));
     let mut result = SharedTensor::<f32>::new(&(1, 1, 3));
+    let payload: &[f32] = &::std::iter::repeat(1f32).take(x.capacity()).collect::<Vec<f32>>();
+    let native = Backend::<Native>::default().unwrap();
+    write_to_memory(x.write_only(native.device()).unwrap(), payload); // Write to native host memory.
+    // FIXME: Add in Sigmoid example for native
+}
+
+#[cfg(feature = "cuda")]
```

Comment:
it might be better to have 
```rust
#[cfg(feature = "cuda")]
type  BackendUnderTest = Cuda;

#[cfg(not(feature = "cuda"))]
type  BackendUnderTest = Native;
```

but keep the rest equiv? Or even go for a `fn run::<B: Backend> (backend: B) {}` so most of the code can be kept common.